typescript/infra/config/environments/mainnet3/warp/addresses.json

```diff
@@ -1,5 +1,9 @@
 {
-  "inevm": {
-    "router": "0x26f32245fCF5Ad53159E875d5Cae62aEcf19c2d4"
+  "ethereum": {
+    "HypERC20Collateral": "0x31Dca7762930f56D81292f85E65c9D67575804fE",
+    "router": "0x31Dca7762930f56D81292f85E65c9D67575804fE"
+  },
+  "viction": {
+    "HypERC20": "0xea820f9BCFD5E16a0dd42071EB61A29874Ad81A4"
```

Comment:
Does this get overwritten after each deploy? If so I'm not sure why it's checked in

Reply:
https://www.notion.so/hyperlanexyz/Runbook-83c755f2652943289cf98cd1309487a8?pvs=4#88352ca0ff634f8c9d7900785da2ad58
its checked in because we often want the tooling to cache from here